frontend/src/app/animals-list/animal-filter/animal-filter.component.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../../assets/styles/variables.scss';
+
+.animal-filter {
+  height: 95px;
+  font-size: 20px;
+
+  &__select,
+  &__select--type{
```

Comment:
Add both classes (I mean `animal-filter__select` and `animal-filter__select-type`) to target html-element.
In that case, styles will look like this:
`
&__select { ... }
&__select-type { ... }
`